package.json

```diff
@@ -30,11 +26,11 @@
     "gulp-eslint": "1.0.0"
   },
   "dependencies": {
-    "byline": "^4.2.1",
-    "compose-function": "^2.0.0",
-    "debug": "^2.2.0",
-    "deepmerge": "^0.2.10",
-    "influent": "0.4.0",
+    "byline": "4.2.1",
+    "debug": "2.2.0",
+    "empty-port": "0.0.1",
+    "fxos-device-service": "4.0.0",
```

Comment:
It wasn't 4.1.1 when I opened the PR, but we should definitely upgrade. Vorpal 1.4.1 is out now with the fix to the missing required options.